ale/drivers/vo_drivers.py

```diff
@@ -6,7 +6,19 @@
 from ale.base.type_sensor import Framer
 from ale.base.base import Driver
 
-class Voyager2IssnacIsisLabelNaifSpiceDriver(Framer, IsisLabel, NaifSpice, Driver):
+class VoyagerCameraLabelNaifSpiceDriver(Framer, IsisLabel, NaifSpice, Driver):
+
+    @property
+    def instrument_id(self):
+        sc_lookup = {
+        "VOYAGER_1" : "VG1",
+        "VOYAGER_2" : "VG2"
+        }
+        sensor_lookup = {
+        "NARROW_ANGLE_CAMERA" : "ISSNA",
+        "WIDE_ANGLE_CAMERA" : "ISSWA"
+        }
+        return sc_lookup[super().spacecraft_name] + '_' + sensor_lookup[super().instrument_id]
```

Comment:
Note for people writing ISIS drivers. You don't need this for ISIS support, but this helps prevent load from accidentally choosing these. I had issues where load picked this for MRO data!

Reply:
How would it have picked the voyager driver for MRO data?

Reply:
I'm guessing it made the driver fine but couldn't format it with the to_isis or to_usgscsm formater?

Reply:
The voyager driver worked with an MRO image and MRO kernels because it's very generic. It actually made it all the way through the formatter.

Reply:
loads calls driver.instrument_id before loading kernels and such as a quick check. This function ensures that unless the spacecraft is actually Voyager it will fail then.